liquibase-maven-plugin/pom.xml

```diff
@@ -45,22 +68,14 @@
             <scope>provided</scope>
         </dependency>
 
-        <dependency>
-            <groupId>org.apache.maven</groupId>
-            <artifactId>maven-artifact</artifactId>
-            <version>${targetMavenVersion}</version>
-            <scope>provided</scope>
-        </dependency>
```

Comment:
Why was this removed?

Reply:
same for this . I think it was done as a part of this commit -> [here](https://github.com/liquibase/liquibase/commit/868d09e7868873358f22ad6127f82fbd1620dc81#diff-56e64c6e32ac9a2a4dd259dd9c572ef3342344f575bc38296d5647c7f5b2c341) to flatten the pom